content/_guides/creating-recipes.md

```diff
@@ -5,166 +5,87 @@ section: Recipes
+
+You can change the visibility using the API function [Update a recipe visibility](#update-a-recipe-visibility):
 
-**EXAMPLE:**
-​
 ```
-curl -X POST {{site.data.tenant.apiBaseUri}}/v2/recipes \
+curl {{site.data.tenant.apiBaseUri}}.io/v2/recipes/{RECIPE_ID}/visibility \
+  -X PATCH \
   -u {EMAIL}:{APIKEY} \
   -H 'Accept: application/json' \
   -H 'Content-Type: application/json' -d '
-  {
-    "data": {
-      "type": "recipe",
-      "attributes": {
-        "activation_config": {
-          "variables": [{
-            "title": "Email to fill a \"CC\" field",
-            "key": "cc"
-          }],
-          "credentials": [{
-            "description": "Credentials to access your Component",
-            "stepId": "step_1"
-          }]
-        },
-        "marketplace_content": {
-          "title": "My Recipe",
-          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-          "short_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-          "help_text": "No setup required",
-          "tags": []
-        },
-        "flow_template": {
-          "cron": "*/3 * * * *",
-          "graph": {
-            "nodes": [
-              {
-                "name": "Step name",
-                "description": "Step description",
-                "command": "acmecore/petstore:getPetsByStatusWithGenerators@latest",
-                "fields": {
-                  "status": "pending"
-                },
-                "id": "step_1"
-              },
-              {
-                "name": "Step name",
-                "description": "Step description",
-                "command": "acmecore/email:send@latest",
-                "fields": {
-                  "dontThrowErrorFlg": true
-                },
-                "id": "step_2"
-              }
-            ],
-            "edges": [
-              {
-                "config": {
-                  "mapper_type": "jsonata",
-                  "mapper": {
-                    "to": "pets[0].name",
-                    "cc": "$getFlowVariables().cc",
-                    "subject": "pets[0].id",
-                    "textBody": "pets[0].status"
-                  },
-                  "condition": null
-                },
-                "source": "step_1",
-                "target": "step_2"
-              }
-            ]
-          }
-        }
-      },
-      "relationships": {
-        "workspace": {
-          "data": {
-            "type": "workspace",
-            "id": "{WORKSPACE_ID}"
-          }
-        }
-      }
-    }
-  }'
+   {
+     "data": {
+       "visibility": "contract"
+     }
+   }'
 ```
 
-### Creating Recipes from Scratch
+> The **"visibility"** field can take the following values: **"workspace"**, **"contract"** or **"tenant"**.
```

Comment:
Workspace level means that Recipe available to be activated into current Workspace
Contract level means that Recipe available to be activated into any Workspace of current Contract
Tenant level means that Recipe available to be activated into any Contract of current Tenant.

Note: you can set visibility level of Recipe according to the Components visibility which used in Recipe.
Note: you can't downgrade Recipe visibility level

Reply:
done